packages/iTwinUI-react/src/core/Wizard/Step.tsx

```diff
@@ -18,11 +18,11 @@ export type StepProps = {
    */
   index: number;
   /**
-   * the Wizard's current step number, 0-based.
+   * the Stepper's current step number, 0-based.
    */
-  currentStepNumber: number;
+  currentStepNumber?: number;
```

Comment:
Why it became optional?

Reply:
I made this optional since the parameter was not being used whenever the Wizard type is workflow, so I made it optional so I didn't have to pass anything for `currentStepNumber` when calling `Step` in `WorkflowDiagram.tsx`. I can change this back and just call `currentStepNumber={0}` for `WorkflowDiagram`?

Reply:
I see you undone this change, so I resolve this comment.